docs/sdk/dev/extend-the-sdk.md

```diff
@@ -5,13 +5,14 @@ template: howto-guide-template
 redirect_from:
     - /docs/sdk/dev/extending-the-sdk.html
 ---
+# SDK development
 
 The SDK offers different extension points to enable third parties to contribute to the SDK without modifying it.
 
 From simple to complex, the SDK can be extended by:
 
-- Providing additional [tasks](/docs/sdk/dev/task.html) or [settings](/docs/sdk/dev/settings.html) via a YAML definition placed inside `<path/to/spryker/sdk>/extension/<YourBundleName>/Task/<taskname>.yaml`. Those tasks can't introduce additional dependencies and are best suited to integrating existing tools that come with a standalone executable.
-- Providing additional tasks, [value resolvers](/docs/sdk/dev/value-resolvers.html), or settings via a PHP implementation placed inside `<path/to/spryker/sdk>/extension/<YourBundleName>/Task/<taskname>.php`. Those tasks need to implement the [TaskInterface](https://github.com/spryker-sdk/sdk-contracts/blob/master/src/Entity/TaskInterface.php) and need to be exposed by providing a Symfony bundle to the Spryker SDK, such as `<path/to/spryker/sdk>/extension/<YourBundleName>/<YourBundleName>Bundle.php`, following the conventions of a [Symfony bundle](https://symfony.com/doc/current/bundles.html#creating-a-bundle). This approach is best suited for more complex tasks that don't require additional dependencies, for example validating the content of a YAML file by using Symfony validators.
+Providing additional [tasks](/docs/sdk/dev/task.html) or [settings](/docs/sdk/dev/settings.html) via a YAML definition placed inside `<path/to/spryker/sdk>/extension/<YourBundleName>/Task/<taskname>.yaml`. Those tasks can't introduce additional dependencies and are best suited to integrating existing tools that come with a standalone executable.
```

Comment:
shouldn't this and the next paragraphs be list items, as the last one in like 16?

Reply:
I'm unsure what you mean here, this is 16 right?:

"- Providing additional tasks, value resolvers, or settings that come with additional dependencies. This approach follows the same guideline as the previous approach with the PHP implementation but requires building  your own [SDK docker image](/docs/sdk/dev/building-flavored-spryker-sdks.html) that includes those dependencies."

What's the difference that 14 should have and 16 has?

Reply:
I mean you have removed "-" from paragraphs 13 and 14, which makes them plain text, and not list items. But looks like they should still be the list items, and not plain text